### ISSUE TITLE
Why isn't caption returned in datasets?

### ISSUE THREAD

**Issue by bhosalems** [hvcl/CoSyn-ContextCondNucAug#2]
For example in below Pannuke dataset

```
    def __getitem__(self, idx):
        # load image
        try:
            original_pil_image = Image.open(self.images[idx]).convert("RGB")
            original_pil_target = Image.open(self.labels[idx])
        except (OSError, ValueError) as e:
            print(f"An exception occurred trying to load file {self.images[idx]}.")
            print(f"Skipping index {idx}")
            return self.skip_sample(idx)

        # Transforms
        image = ToTensor()(original_pil_image)
        label = ToTensorNoNorm()(original_pil_target).float()
        img_lbl = self.augmentation(torch.cat([image, label]))

        caption = img_lbl[3:] # caption 자리에 뭐라도 넣어야 돌아감

        return img_lbl[:3], img_lbl[3:], caption
```

Why is the caption taken just as a label? Where exactly text be read from ? Is this to avoid dataloader problems that happen if you do not return the same size and type of data as iterables.

**Comment by hjoh01** [comment]
Hi, 
I have updated the repository. I have found that this code was older version with no caption.
